pkg/inject/config.go

```diff
@@ -55,6 +55,8 @@ const (
 	flagXRayImage            = "xray-image"
 
 	flagClusterName = "cluster-name"
+	flagUseAwsFipsEndpoint = "use-aws-fips-endpoint"
+	flagUseAwsDualStackEndpoint    = "use-aws-dual-stack-endpoint"
```

Comment:
The formatting here doesn't quite look right - can you run this through gofmt? (e.g. `gofmt -w pkg/inject/config.go`). See [here](https://go.dev/blog/gofmt) for some details.

Reply:
The makefile runs go fmt in the all and cotroller target. Are we not using the makefile for builds? If we're not we should figure out why.

Reply:
I think make file runs go fmt. I have had merge conflicts when I am syncing my fork with main repo. may be that is the reason.